### DISCUSSION TITLE
How can I load light and dark favicons?

### DISCUSSION THREAD

**Discussion by babakfp** [vuejs/vitepress#3301] [Q&A] [answered]
.

**Comment by brc-dd** [comment, marked as answer]
I think the recommended way is to use svg favicons - https://stackoverflow.com/a/60158619/11613622 -- link tag in vitepress can be added like this - https://vitepress.dev/reference/site-config#head

**Reply by babakfp** [reply]
Thanks for your help.
I made this: https://github.com/vuejs/vitepress/discussions/3304
